frontends/mit-open/src/pages/DepartmentListingPage/DepartmentListingPage.tsx

```diff
@@ -0,0 +1,232 @@
+import React from "react"
+import {
+  Container,
+  Typography,
+  styled,
+  PlainList,
+  List,
+  ListItem,
+  ListItemLink,
+  ListItemText,
+} from "ol-components"
+import type { TypographyProps } from "ol-components"
+import { MetaTags, pluralize } from "ol-utilities"
+import type {
+  LearningResourceSchool,
+  LearningResourceSearchResponse,
+} from "api"
+import {
+  useLearningResourcesSearch,
+  useSchoolsList,
+} from "api/hooks/learningResources"
+import {
+  RiPaletteLine,
+  RiSeedlingLine,
+  RiBriefcaseLine,
+  RiMacbookLine,
+  RiBarChartBoxLine,
+  RiUserSearchLine,
+  RiArrowRightSLine,
+} from "@remixicon/react"
+
+const SCHOOL_ICONS: Record<string, React.ReactNode> = {
+  "https://sap.mit.edu/": <RiPaletteLine />,
+  "https://engineering.mit.edu/": <RiBriefcaseLine />,
+  "https://shass.mit.edu/": <RiMacbookLine />,
+  "https://science.mit.edu/": <RiBarChartBoxLine />,
+  "http://mitsloan.mit.edu/": <RiSeedlingLine />,
+  "https://computing.mit.edu/": <RiUserSearchLine />,
+}
+
+const FullWidthBackground = styled.div`
+  background-image: url("/static/images/background_steps.jpeg");
+  background-size: cover;
+  padding-top: 48px;
+  padding-bottom: 48px;
+  color: ${({ theme }) => theme.custom.colors.white};
```

Comment:
Is the theme ever intended to be reactive (e.g. dark/light toggle). If not we can just import and do 
```
color: ${theme.custom.colors.white};
```
... though that would create more work if we ever introduced reactive theming.

Reply:
I don't have a strong preference one way or the other, the function version always seemed easy since I didn't need to import the theme :shrug:.